[BUG] Do not use backticks in shell files

### Is there an existing issue for this?

- [x] I have searched the existing issues.

### Current Behavior

[gh_retry.sh](https://github.com/projectdiscovery/nuclei/blob/d1614857bd345affb24f209aeebde94db7303b6d/gh_retry.sh) uses depreciated _backticks_ (cf. [SC2006](https://www.shellcheck.net/wiki/SC2006)).

### Expected Behavior

Not relevant.

### Steps To Reproduce

One can run [shellcheck](https://www.shellcheck.net/) on mentioned script.

### Relevant log output

Not relevant.

### Environment

Not relevant.

### Anything else?

Feel free to assign me to this issue.

Usually I would submit a PR directly, but the [contribution guidelines](https://github.com/projectdiscovery/nuclei/blob/d1614857bd345affb24f209aeebde94db7303b6d/CONTRIBUTING.md) asked to create an issue first.

--- a/gh_retry.sh
+++ b/gh_retry.sh
@@ -21,9 +21,9 @@ WORKFLOW="Build Test"
 GREP_ERROR_PATTERN='Test "http/interactsh.yaml" failed'
 
 #Set fonts for Help.
-NORM=`tput sgr0`
-BOLD=`tput bold`
-REV=`tput smso`
+NORM=$(tput sgr0)
+BOLD=$(tput bold)
+REV=$(tput smso)
 
 HELP()
 {
@@ -73,7 +73,7 @@ function print_bold() {
 function retry_failed_jobs() {
     print_bold "Checking failed workflows for branch $BRANCH before $BEFORE"
 
-    date=`date +%Y-%m-%d'T'%H:%M'Z' -d "$BEFORE"`
+    date=$(date +%Y-%m-%d'T'%H:%M'Z' -d "$BEFORE")
 
     workflowIds=$(gh run list --limit "$LIMIT"  --json headBranch,status,name,conclusion,databaseId,updatedAt | jq -c '.[] |
     select ( .headBranch==$branch ) |

